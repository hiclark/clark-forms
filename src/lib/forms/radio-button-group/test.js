import React from 'react';

import RadioButtonGroup from './';

const OPTIONS = [
  { value: 1, label: 'List Item 1' },
  { value: 2, label: 'List Item 2' },
  { value: 3, label: 'List Item 3' },
  { value: 4, label: 'List Item 4' },
  { value: 5, label: 'List Item 5' },
];

describe('<RadioButtonGroup />', () => {
  let wrapper;

  beforeEach(() => {
    const props = {
      name: 'radioButton',
      type: 'radioButton',
      values: OPTIONS,
    };
    wrapper = shallow(<RadioButtonGroup {...props} />);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
