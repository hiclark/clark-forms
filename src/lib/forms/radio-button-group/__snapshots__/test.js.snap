// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RadioButtonGroup /> matches the snapshot 1`] = `
<Field
  component={[Function]}
  name="radioButton"
  type="radioButton"
  values={
    Array [
      Object {
        "label": "List Item 1",
        "value": 1,
      },
      Object {
        "label": "List Item 2",
        "value": 2,
      },
      Object {
        "label": "List Item 3",
        "value": 3,
      },
      Object {
        "label": "List Item 4",
        "value": 4,
      },
      Object {
        "label": "List Item 5",
        "value": 5,
      },
    ]
  }
/>
`;
